apis/supernova/src/generator.js

```diff
@@ -73,7 +74,9 @@ export default function generatorFn(UserSN, galaxy) {
       const ss = create(generator, params, galaxy);
       return ss;
     },
-    definition: {},
+    definition: {
+      dataHandler: (opts) => new HyperCubeHandler(opts),
+    },
```

Comment:
How do we solve this later:
https://github.com/qlik-trial/nebula-tables/blob/main/packages/sn-pivot-table/src/ext/index.ts#L17
We will be moving them into each respective chart, so how do we make sure they get picked up correctly? Will that be done here somewhere as well? (later)

Reply:
Also think we want to flag this, to avoid issues and keep it limited for the time being.

Reply:
In case of the pivot table, there is actually a task to move the related handlers to nebula-table so I guess they will be handled there, haven't looked into it yet but I'll soon.

Yes, the flag is on the way, don't have access to create by myself but have asked for that.

Reply:
In case of pivot table, there is actually a task to move the related handlers to nebula-table so I guess they will be handled there, haven't looked into it yet but I'll soon.

Reply:
@Caele Btw, do you mean we put the feature flag in Nebula as an open-source repo? it's a question @enell came up with.